soccer/Robot.hpp

```diff
@@ -187,7 +215,14 @@ class OurRobot: public Robot {
 	 * @brief Move to a given point using the default RRT planner
 	 * @param endSpeed - the speed we should be going when we reach the end of the path
 	 */
-	void move(const Geometry2d::Point &goal, float endSpeed = 0);
+	void move(const Geometry2d::Point &goal, Geometry2d::Point endVelocity = Geometry2d::Point());
+
+
+	/**
+	 * @brief Move to a given point using the default RRT planner
```

Comment:
The point of moveDirect is that it doesn't use the planner, right?  If so, let's update this comment.